lib/WebpackOptionsValidationError.js

```diff
@@ -362,6 +362,10 @@ class WebpackOptionsValidationError extends WebpackError {
 					err.parentSchema
 				)}`;
 			}
+		} else if (err.keyword === "not") {
+			return `${dataPath} should not be ${getSchemaPartText(
+				err.schema
+			)}\n${getSchemaPartText(err.parentSchema)}`;
```

Comment:
We should port this in `schema-utils`, right not is not implemented

Reply:
yep